certified-connectors/Powell Teams/README.md

```diff
@@ -0,0 +1,24 @@
+# Powell Teams
+
+Powell Teams API endpoints documentation.

```

Comment:
@PowellSoftware Can you please provide more information about your connector, instead of just mentioning documentation.